docs/ops/arithmetic/Mod_1.md

```diff
@@ -10,7 +10,7 @@
 As a first step input tensors *a* and *b* are broadcasted if their shapes differ. Broadcasting is performed according to `auto_broadcast` attribute specification. As a second step *Mod* operation is computed element-wise on the input tensors *a* and *b* according to the formula below:
 
 \f[
-o_{i} = a_{i} % b_{i}
+o_{i} = a_{i} \mod b_{i}
```

Comment:
old:

![image](https://user-images.githubusercontent.com/39924818/128856738-19b63af1-af47-464b-9c8d-984fd17b035d.png)

new:

![image](https://user-images.githubusercontent.com/39924818/128856763-88c93388-a785-4104-9b29-caf20c397277.png)

Here old mod sign was not rendered at all. There's exactly the same problem with FloorMod, I'll update it in next commit.